src/main/kotlin/eu/kevin/api/models.kt

```diff
@@ -0,0 +1,6 @@
+package eu.kevin.api
```

Comment:
my file is named **models.kt** instead of **Authorization.kt**? same question to the files above

Reply:
It might be easier to read the code when related simple classes are in the same file (https://kotlinlang.org/docs/coding-conventions.html#source-file-names, https://kotlinlang.org/docs/coding-conventions.html#source-file-organization)

Reply:
"Encouraged" does not mean recommended, especially when there is a paragraph above stating that files should be named in the same way the containing classes are. Also in this case models is a very abstract term where you can put any model entity or DTO, so this is likely to grow quickly. During the initial development stages I would start with as much separation as possible.